<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="settingsgbox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="workLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set work time in minutes, every &lt;span style=&quot; font-weight:600;&quot;&gt;n&lt;/span&gt; minutes you will have a long or short break depending on your settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Work&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="workSpinBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set work time in minutes, every &lt;span style=&quot; font-weight:600;&quot;&gt;n&lt;/span&gt; minutes you will have a long or short break depending on your settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="suffix">
           <string>min</string>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sbreakLabel">
          <property name="toolTip">
           <string>Set the short break time in minutes, each time a work period is complete you will have a break, either long or short depending on your settings</string>
          </property>
          <property name="text">
           <string>Short break</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbreakSpinBox">
          <property name="suffix">
           <string>min</string>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbreakLabel">
          <property name="toolTip">
           <string>Set the long break time in minutes, each time a work period is complete you will have a break, either long or short depending on your settings</string>
          </property>
          <property name="text">
           <string>Long break</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lbreakSpinBox">
          <property name="toolTip">
           <string>Set the short break time in minutes, each time a work period is complete you will have a break, either long or short depending on your settings</string>
          </property>
          <property name="suffix">
           <string>min</string>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="toolTip">
           <string>Set the short break time in minutes, each time a work period is complete you will have a break, either long or short depending on your settings</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Early&lt;br&gt;notification&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="earlynotifySpinBox">
          <property name="toolTip">
           <string>If you wish to receive a notification before the work time is over specify the time in seconds that you want your early notification</string>
          </property>
          <property name="suffix">
           <string>sec</string>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
